application-development/gae-microservice/server/goapp.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2017 Google Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This program is a web application that accepts a large POST request
+// and returns a large response to demonstrate and test streaming
+// data in a microservice. The web application optionally sets the
+// X-Accel-Buffering header to no to avoid buffering to disk in the
+// NGINX server used in Flex. For detai
```

Comment:
truncated comment... also  it should say "to avoid buffering to disk" 

thanks